src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Random random = new Random();
+        int randomIndex = random.nextInt(Color.values().length);
+
+        return  Color.values()[randomIndex].toString();
     }
 }
```

Comment:
Classes, fields, constructors, and methods should have access modifiers to specify their visibility. In this case, the class and method should be marked as `public` if they are intended to be used outside of the package.